### PR TITLE
JDP230102-25 implemented methods for user controller

### PR DESCRIPTION
**Pull request by Feliscatus8**

### CODE REVIEW

**Review by DawidNst — APPROVED** [review]

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java [hunk 1]
```diff
@@ -10,14 +11,17 @@
...
 @CrossOrigin("*")
 public class UserController {
+    private UserDbService userDbService;
 
     @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<UserAddDto> addUser(@RequestBody UserAddDto userAddDto) {
+    public ResponseEntity<UserAddDto> addUser(@RequestBody UserAddDto userAddDto) throws Exception {
+        userDbService.saveUser(userAddDto);
         return ResponseEntity.ok(userAddDto);
     }
```
Comment (gawrysiakg) [inline src/main/java/com/kodilla/ecommercee/controller/UserController.java:20]: I think ... it should return Void. Returning the same dto with username and password its not a good idea

**Review by gawrysiakg — COMMENTED** [review]
I think we all have a problem. There is no endpoint here: generating a random key valid one hour after entering the correct user data.
We moved it from here to session controller but it wasn't made by anyone, it's a very serious bug. All in all, it's not good that we moved him, as it was. It should take userAddDto and in response entity return loginDto or sessionDto
    "loggedIn": true,
    "sessionId": 12345

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java (hunk 1 above)
Comment (DawidNst) [reply src/main/java/com/kodilla/ecommercee/controller/UserController.java:20]: I was wondering about that too

Comment (Feliscatus8) [reply src/main/java/com/kodilla/ecommercee/controller/UserController.java:20]: true, i didn't notice this problem

**Review by MMacina — APPROVED** [review]

**Pull request merged**